ui/components/multichain/connected-status/connected-status.tsx

```diff
@@ -43,11 +43,11 @@ export const ConnectedStatus: React.FC<ConnectedStatusProps> = ({
     status = STATUS_CONNECTED_TO_ANOTHER_ACCOUNT;
   }
 
-  let badgeBorderColor = BackgroundColor.backgroundDefault; // TODO: Replace it once border-color has this value.
-  let badgeBackgroundColor = Color.borderMuted; // //TODO: Replace it once Background color has this value.
+  let badgeBorderColor = BorderColor.backgroundDefault; // TODO: Replace it once border-color has this value.
+  let badgeBackgroundColor = BackgroundColor.iconAlternative;
```

Comment:
We recently updated border muted to be transparent which is why the UI looked "broken" I've updated it to use icon/alternative. This also matches Figma

<img width="383" alt="Screenshot 2024-12-19 at 7 41 54 AM" src="https://github.com/user-attachments/assets/8e2d0eb7-d0c7-4d75-97f4-687e5063d473" />
<img width="372" alt="Screenshot 2024-12-19 at 7 42 08 AM" src="https://github.com/user-attachments/assets/16d27769-af65-46d1-87a0-4bce53c743b3" />
<img width="1073" alt="Screenshot 2024-12-19 at 7 43 48 AM" src="https://github.com/user-attachments/assets/f33465e2-86b2-43a1-8018-24de41be5041" />